reflex-dom-contrib.cabal

```
@@ -62,18 +62,18 @@ library
      Reflex.Dom.Contrib.Widgets.ScriptDependent
 
   build-depends:
-    aeson                  >= 0.8    && < 1.3,
-    base                   >= 4.6    && < 4.11,
+    aeson                  >= 0.8    && < 1.5,
+    base                   >= 4.6    && < 5.0,
```

Can you change this to 4.13 instead of 5.0?  Other than that, looks great!

```suggestion
    base                   >= 4.6    && < 4.13,
```